CONTRIBUTING.md

```diff
@@ -62,26 +70,45 @@ Tests also depend on NeTEx specification that are imported as a git submodule.
 Therefore, these tests are run only if feature `xmllint` is activated.\
 
 To install xmllint and submodules:
+
 ```sh
 git submodule update --init --recursive
 apt install libxml2-utils
 ```
 
-#### Launch all tests
+#### Check outputs manually
+
+To validate the NeTEx output it is possible to use xmllint:
 
 ```sh
-# Run all the tests of `transit_model` in the entire repository,
-# activating all features, including `xmllint`
-cargo test --workspace --all-features
+xmllint --noout --nonet --huge --schema /path/to/NeTEx/xsd/NeTEx_publication.xsd your_file.xml
 ```
 
-#### Hand-check outputs
+Note: this may be very (very) slow on huge files.
+
+#### Launch all tests
 
-To validate the output NeTEx obtained it is possible to use xmllint:
 ```sh
-xmllint --noout --nonet --huge --schema /path/to/NeTEx/xsd/NeTEx_publication.xsd your_file.xml
+# Run all the tests of `transit_model` in the entire repository,
+# without fixtures, then activating all features, including `xmllint`
+# to make sure that both works
+cargo test --workspace --all-features && cargo test --workspace
```

Comment:
If I understand your previous sentence above it should be

```suggestion
cargo test --workspace && cargo test --workspace --all-features 
```

Reply:
Haha, it was that, then @pbougue thought it'd be better to run `--all-features` first to get the full compile first and get feedback faster. I guess he forgot to update the comment above.